docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -169,6 +169,12 @@ quarkus.keycloak.policy-enforcer.enable=true
 
 NOTE: By default, applications using the `quarkus-oidc` extension are marked as a `service` type application (see `quarkus.oidc.application-type`). This extension currently supports only such `service` type applications.
 
+=== Configuring protected routes
+
+In the default configuration, Keycloak is responsible for managing the roles and deciding who can access which routes. 
+
+To configure the protected routes using the `application.properties` file, check the link:security-openid-connect-web-authentication[Protecting Web Applications Using OpenID Connect] guide. For more details, check the link:security[Security guide].
+
```

Comment:
Hi, FYI, `quarkus-keycloak-authorization` is not activated for the `web-app` applications which is what this section refers to. Can you please clarify what do you mean when you say `the protected routes` ?

Reply:
`the protected routes` refers to the URIs such as this one https://github.com/quarkusio/quarkus-quickstarts/blob/1fcd29b58ba3a80bf25ce3f5dab8d02ad0a56226/security-keycloak-authorization-quickstart/config/quarkus-realm.json#L477

Reply:
@stefnotch Thanks. I'm sorry but I don't understand the new document addition. `quarkus-keycloak-authorization` is to support the OIDC `service` applications only when (as per your PR, Keycloak does the policy-driven authorization). If one has an OIDC `web-app` application which your PR links to then this `quarkus-keycloak-authorization` will not be activated.
This is why I'm confused about the message of this PR. Perhaps I'm missing the point :-), please clarify. CC @pedroigor 

Thanks

Reply:
@stefnotch Perhaps you meant to link to `security-openid-connect` ?

Reply:
I see, I indeed missed `quarkus.oidc.application-type=web-app`. Thank you for catching it, I'll fix it.

Reply:
@stefnotch Thank you :-)